Mathlib/Algebra/Order/Floor.lean

```diff
@@ -677,9 +677,7 @@ theorem floorRing_ceil_eq : @FloorRing.ceil = @Int.ceil :=
 
 /-! #### Floor -/
 
-
--- Porting note: changed `(coe : ℤ → α)` to `(Int.cast : ℤ → α)`
-theorem gc_coe_floor : GaloisConnection (Int.cast : ℤ → α) floor :=
+theorem gc_coe_floor : GaloisConnection ((↑) : ℤ → α) floor :=
   FloorRing.gc_coe_floor
 #align int.gc_coe_floor Int.gc_coe_floor
```

Comment:
@YaelDillies, do you know why we have both of these lemmas? Was one about a different coercion in mathlib3?

Reply:
I think it was just for discoverability as the operator is `Int.floor`, not `FloorRing.floor`.